### PR TITLE
Update flp2epn-distributed docs.

### PR DESCRIPTION
**Pull request by rbx**

### CODE REVIEW

File: Examples/flp2epn-distributed/src/EPNReceiver.cxx [hunk 1]
```diff
@@ -148,6 +144,12 @@ void EPNReceiver::Run()
             LOG(ERROR) << "Could not send acknowledgement without blocking";
           }
         }
+        else
```
Comment (matthiasrichter) [inline Examples/flp2epn-distributed/src/EPNReceiver.cxx]: What is the reason for this change? It's not documentation related.

Comment (rbx) [inline Examples/flp2epn-distributed/src/EPNReceiver.cxx]: This is to avoid sending when nothing is connected, as it is in the test mode. Does it break your setup?

**Comment by sawenzel** [comment]
The changes are ok for me. I guess that the new EPN developments (@ktf, @matthiasrichter, @sawenzel) should then not go here (into this example) but directly into a more production directory.

**Comment by matthiasrichter** [comment]
yes, the only concern was that this change interferes with the recent update in some forks. But it's better to make a production version from that when it is running and leave this example as it is. So there is not more potential conflict.

File: Examples/flp2epn-distributed/README.md [hunk 2]
```diff
@@ -3,62 +3,50 @@
 
 #### General
 
-The devices implement following topology:
+The devices implement following topology:
```
Comment (matthiasrichter) [inline Examples/flp2epn-distributed/README.md]: very minor thing: there is an accidentally inserted blank
